camel-idea-plugin/src/main/java/org/apache/camel/idea/annotator/CamelEndpointAnnotator.java

```diff
@@ -201,31 +201,7 @@ public String getErrorMessage(EndpointValidationResult result, Map.Entry<String,
     private static class UnknownErrorMsg implements CamelAnnotatorEndpointMessage<String> {
         @Override
         public String getErrorMessage(EndpointValidationResult result, String property) {
-            // for each invalid option build a reason message
-            String returnMsg = "";
-            if (result.getUnknown() != null) {
-                for (String name : result.getUnknown()) {
-                    if (name.equals(property)) {
-                        if (result.getUnknownSuggestions() != null && result.getUnknownSuggestions().containsKey(name)) {
-                            String[] suggestions = result.getUnknownSuggestions().get(name);
-                            if (suggestions != null && suggestions.length > 0) {
-                                String str = Arrays.asList(suggestions).toString();
-                                returnMsg += "\nUnknown option. Did you mean: " + str;
-                            } else {
-                                returnMsg += "\nUnknown option";
-                            }
-                        } else {
-                            returnMsg = "Unknown option";
-                        }
-                    }
-                }
-            }
-            if (result.getRequired() != null) {
-                for (String name : result.getRequired()) {
-                    returnMsg += "\nMissing required option";
-                }
-            }
-            return returnMsg;
+            return "Unknown option";
```

Comment:
This was not enabled, see this ticket: https://github.com/davsclaus/camel-idea-plugin/issues/156